libs/application/templates/general-fishing-license/src/forms/GeneralFishingLicenseForm/applicantInformationSection.ts

```diff
@@ -44,8 +44,8 @@ export const applicantInformationSection = buildSection({
           disabled: true,
           required: true,
           defaultValue: (application: GeneralFishingLicense) =>
-            application.externalData?.nationalRegistry?.data?.address
-              ?.streetAddress,
+            application.externalData.identityRegistry.data.address
```

Comment:
Do we need the optional chaining operator to prevent error if address is undefined?